devnotes/pnor_layout.md

```diff
@@ -0,0 +1,186 @@
+| HB_VOLATILE                 | 0x1B | 0x03CE9000:0x03CEDFFF  | 0x5000       | d    | -     | same as size | None                                  |
+| [MEMD](#MEMD)               | 0x1C | 0x03cee000:0x03CFBFFF  | 0xE000       | d    | -     | same as size | VPD for RAM IIRC                      |
+| SBKT                        | 0x1D | 0x03d02000:0x03D05FFF  | 0x4000       | d    | -     | same as size | None                                  |
+| [HDAT](#HDAT)               | 0x1E | 0x03d06000:0x03D0DFFF  | 0x8000       | d    | -     | same as size | None                                  |
+| UVISOR                      | 0x1F | 0x03d10000:0x03E0FFFF  | 0x100000     | d    | -     | same as size | None                                  |
+| [BOOTKERNFW](#BOOTKERNFW)   | 0x20 | 0x03e10000:0x03FEFFFF  | 0x1E0000     | d    | -     | same as size | None                                  |
+| [BACKUP_PART](#BACKUP_PART) | 0x21 | 0x03ff7000:0x03FFEFFF  | 0x8000       | d    | -     | 0            | Required                              |
+
+> Note: If partition has no coreboot usage means that erasing this partition
+> didn't prevent the platform from starting skiroot.
+> Partition may still be used in some case
+> or in the later stages of the booting process.
+
+Type:
+* l = FFS_TYPE_LOGICAL
+* d = FFS_TYPE_DATA
+* p = FFS_TYPE_PARTITION
+
+Flags:
+* b = FFS_FLAGS_U_BOOT_ENV
+* p = FFS_FLAGS_PROTECTED
+
+# Reproducibility
+Reproducibility of partitions can be found at the
+[wiki page](https://wiki.raptorcs.com/wiki/Firmware/Reproducible_Builds/Status).
+
+# Partition original usage
+## part
+This partition contains partition table.
+
+## HBEL
+This partition is used by `hostboot` to to save crash information for later
+debug.
+More informations can be found at the
+[wiki page](https://wiki.raptorcs.com/wiki/Hostboot_Debug_Howto).
+
+## GUARD
+This partition is responsible for holding information about components that
+are detected as broken.
+More information about it can be found at the
+[wiki page](https://wiki.raptorcs.com/wiki/Troubleshooting/Guard_Partition).
+
+## NVRAM
+Non volatile memory required for OPAL systems.
+More informations can be fouind on the
+[wiki page](https://open-power.github.io/skiboot/doc/opal-api/opal-nvram-read-write-7-8.html).
+
+## SECBOOT
+This partition is used to hold secure variables across renppts in
+tamper-resistant manner. Because writes to PNOR can't be prevented, hash of
+this partition is stored in the `TPM NV`, and verified at the boot time.
+More informations can be found at the
+[wiki page](https://open-power.github.io/skiboot/doc/secvar/secboot_tpm.html).
+
+## HBB
+Bootloader code for `hostboot`
+
+## HBD
+Hostboot data
+
+## HBI
+Main `hostboot` application
+
+## SBE
+Self Boot Engine, stored in module EEPROM
+
+## HCODE
+FW for PPE
+
+## PAYLOAD
+This partition contains `skiboot`.
+
+## BOOTKERNEL
+This partition contains `skiroot`, `petitboot` and initramfs.
```

Comment:
This partition contains `skiroot`: Linux kernel bundled with `initrd` (Linux userspace and  `petitboot` binaries).

Reply:
Fixed